packages/browser/src/plugins/remote-loader/index.ts

```diff
@@ -30,14 +32,18 @@ export class ActionDestination implements DestinationPlugin {
   type: Plugin['type']
 
   alternativeNames: string[] = []
+  options: InitOptions
+
+  private loadPromise = createDeferred<unknown>()
 
   middleware: DestinationMiddlewareFunction[] = []
 
   action: Plugin
 
-  constructor(name: string, action: Plugin) {
+  constructor(name: string, action: Plugin, options: InitOptions) {
```

Comment:
Do we need the whole options object? Can we define instead a subset object of options just for plugins?
I prefer the later since it is easier to limit what we expose, and also to limit the impact if we ever have to change things in the client options.

Reply:
Should also be optional

Reply:
I would also argue that the timeout should be an option only for the `load` method, not for the whole plugin defined at construction time (do we let `.load` be called multiple times? when is the constructor of the plugin called?)